python/tests/test_tables.py

```diff
@@ -1320,7 +1320,7 @@ def test_incompatible_edges(self):
         tables2 = ts2.dump_tables()
         tables2.edges.set_columns(**tables1.edges.asdict())
         # The edges in tables2 will refer to nodes that don't exist.
-        self.assertRaises(_tskit.LibraryError, tables1.sort())
```

Comment:
Jeez, how did this pass! The 
```
with self.assertRaises(xx):
    do_thing()
```
notation is no much better.

Reply:
It seems `None` as the callable just skips the test!! Can see why 3.8 changed that!